packages/frontend/components/composed/contribution/ContributionsBlock/ContributionBlockItem/ContributionBlockItem.styles.ts

```diff
@@ -9,18 +10,32 @@ export const ListItem = styled.li`
   ${respond(`grid-column: span 12;`, 70)}
 `;
 
-export const ItemLink = styled.a`
+export const ItemContent = styled.div`
   display: flex;
 
   > * + * {
     padding-inline-start: ${pxToRem(20)};
   }
 `;
 
-export const ItemAvatar = styled.div`
+export const ItemAvatar = styled.a`
   flex: none;
 `;
 
 export const ItemMetadata = styled.div`
   padding-block-start: var(--padding-xs);
 `;
+
+export const ORCIDLink = styled(ORCIDLinkBase)`
+  ${aLink}
```

Comment:
Since the base includes link styling, this can be removed. Even with that change, the hover is a little odd:
<img width="274" alt="Screen Shot 2022-05-26 at 11 26 49 AM" src="https://user-images.githubusercontent.com/57107444/170552487-1ca5862a-f32c-4a59-b26f-c55600179b24.png">
Because the fancy hover effect is a background, not underline text, the line appears at the bottom of the span block. It's tricky to get right!

Here's the styling with everything but the `svg >` block removed. It looks pretty good, and I'd be OK with it like this, but if we want the flex layout we need to do something like the below option.
<img width="258" alt="Screen Shot 2022-05-26 at 11 32 25 AM" src="https://user-images.githubusercontent.com/57107444/170553571-81dd4ccc-a756-4670-bad7-2f9611e9d9fa.png">

For the flex layout, try updating the atomic component to this, and then style the `a` tag with the flex layout. If we make the change here, we can keep the link display consistent between both pages.
```
/* Simple download text and icon,
 * style can be changed using the className property */
export default function ORCIDLink({ children, ...props }: BaseProps) {
  return children ? (
    // Change `a` to a styled link with flex
    <a target="_blank" {...props}>
      <IconFactory icon="orcid" />
      // The utility class a-link has link styling
      <span className="a-link">{children}</span>
    </a>
  ) : null;
}
```